BlockEntity/BEFruitpress.cs

```diff
@@ -269,23 +274,11 @@ private void onTick100msServer(float dt)
             }
 
 
-            if (juiceableLitresLeft <= 0.01)
+            if (juiceableLitresLeft < 0.01)
             {
-                // Hack to fix rounding errors
-                if (cntBlock != null)
-                {
-                    float litres = cntBlock.GetCurrentLitres(BucketSlot.Itemstack);
-                    cntBlock.SetCurrentLitres(BucketSlot.Itemstack, (float)Math.Round(100 * litres) / 100f);
-                }
-

```

Comment:
This hack is no longer necessary since we are simply rounding the value in litersToTransfer to 2 decimal places up above before it moves it around.